lib/models/settings_model.dart

```diff
@@ -111,11 +127,11 @@ class SettingsModel implements Model {
       'activitiesCount': activitiesCount,
       'theme': theme.index + 1,
       'nrOfDaysToDisplay': nrOfDaysToDisplay,
-      'lockTimerControl' : lockTimerControl,
+      'lockTimerControl': lockTimerControl,
       'greyScale': greyscale,
-      'pictogramText' : pictogramText,
+      'pictogramText': pictogramText,
       'weekDayColors':
-      weekDayColors?.map((WeekdayColorModel e) => e.toJson())?.toList()
+          weekDayColors?.map((WeekdayColorModel e) => e.toJson())?.toList()
```

Comment:
Indentation issue

Reply:
No

Reply:
I understand that you have moved it to a new line because of the 80 character limit, but why is it indented with 4 spaces, while the rest is indented with 2 spaces?

Reply:
That is the autoformatting rules.